chapters/annotations.tex

```diff
@@ -1817,41 +1905,93 @@ \subsubsection{Conversion Rules}\label{conversion-rules}
 This is useful if there is no possibility to convert a specific parameter (or other element), especially if it rarely modified.  If the parameter had no impact on the model it can be removed using \lstinline!convertModifiers!, see \cref{convertmodifiers}.
 \end{nonnormative}
 
+
 \subsection{Versions in the File System}\label{mapping-of-versions-to-file-system}\label{versions-in-the-File-System}
 
-A top-level class, \lstinline!IDENT!, with version \lstinline!VERSION-NUMBER! can be stored in
-one of the following ways in a directory given in the \lstinline!MODELICAPATH!:
+The top-level class $\mathit{packageName}$ with version $\mathit{packageVersion}$ (matching \lstinline[language=grammar]!PACKAGE-VERSION! defined in \cref{version-numbering}) can be stored in one of the following ways in a directory given in the \lstinline!MODELICAPATH! (\cref{the-modelica-library-path-modelicapath}):
 \begin{itemize}
 \item
-  The file \lstinline!IDENT ".mo"!\\
+  The file $\mathit{packageName}$\filename{.mo}\\
   Example: \filename{Modelica.mo}
 \item
-  The file \lstinline!IDENT " " VERSION-NUMBER ".mo"!\\
+  The file $\mathit{packageName}$\textvisiblespace$\mathit{packageVersion}$\filename{.mo} (one space character separating $\mathit{packageName}$ from $\mathit{packageVersion}$)\\
```

Comment:
Just because it is customary to use `\textvisisblespace` where close attention to spaces is needed in the LaTeX world of high quality technical documentation.

Reply:
It was also the resolution of the length conversation ending in https://github.com/modelica/ModelicaSpecification/pull/2957#discussion_r867151745.

Reply:
Well, it was resolved that way due to the lack of other comments. And now we have another comment.

Reply:
Done.  Resolving.